internal/injector/mutate/handler.go

```diff
@@ -32,18 +33,19 @@ func (h *Handler) Mutate(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func (h *Handler) mutate(w http.ResponseWriter, r *http.Request) (int, error) {
+func (h *Handler) mutate(w io.Writer, r *http.Request) (int, error) {
```

Comment:
Why did the signature changed?

Reply:
ah go lint

internal/injector/mutate/handler.go:35:26: w can be io.Writer (interfacer) 

func (h *Handler) mutate(w http.ResponseWriter, r *http.Request)